src/bsoncxx/builder/stream/key_context.hpp

```diff
@@ -36,12 +37,12 @@ class key_context {
 
     template <std::size_t n>
     value_context<key_context> operator<<(const char (&v)[n]) {
-        _core->key_literal(v, n - 1);
+        _core->key_view(stdx::string_view{v, n - 1});
```

Comment:
I suspect we'll want a stdx::string_view overload as well as the literal and std::string options